src/libraries/System.Text.RegularExpressions/gen/RegexGenerator.Emitter.cs

```diff
@@ -371,7 +371,7 @@ private static RequiredHelperFunctions EmitFindFirstChar(IndentedTextWriter writ
             writer.WriteLine();
 
             writer.WriteLine("// No starting position found");
-            writer.WriteLine("ReturnFalse:");
+            writer.WriteLine("NoStartingPositionFound:");
```

Comment:
NIT: since this is used in a few places, should it be stored in a local string instead?

Reply:
Sure, I can move it into a named const.